src/passes/externalInputChecker/enumBoundChecker.ts

```diff
@@ -0,0 +1,67 @@
+import assert = require('assert');
+import { AST } from '../../ast/ast';
+import {
+  FunctionDefinition,
+  Identifier,
+  FunctionVisibility,
+  Statement,
+  ExpressionStatement,
+  VariableDeclaration,
+  FunctionCall,
+} from 'solc-typed-ast';
+import { ASTMapper } from '../../ast/mapper';
+
+export class EnumBoundChecker extends ASTMapper {
+  visitFunctionDefinition(node: FunctionDefinition, ast: AST): void {
+    if (FunctionVisibility.External === node.visibility) {
```

Comment:
In all 3 files, this should include a check that the function has a body, doesn't make sense to try to these checks into undefined functions

Reply:
Added.